web/src/upload.js

```diff
@@ -161,8 +169,9 @@ export async function upload_files(uppy, config, files) {
     if (get_item("markdown_preview_hide_button", config).is(":visible")) {
         get_item("markdown_preview_hide_button", config).trigger("click");
     }
-
+    uploads_in_progress = true;
     get_item("send_button", config).prop("disabled", true);
+    get_item("send_button", config).css("opacity", 0.6);
```

Comment:
I think it would be better to do this in the CSS file instead of here, as a style on the disabled attribute. It can apply overall to the disabled submit button and not only during upload.